docs/website/docs/getting-started/pytorch.md

```diff
@@ -54,21 +52,26 @@ example_input = # ... an input to the model with the expected shape and dtype
 mlir = torch_mlir.compile(
     model,
     example_input,
-    output_type=torch_mlir.OutputType.LINALG_ON_TENSORS,
+    output_type="linalg-on-tensors",
     use_tracing=True)
 ```
 
 The full list of available output types can be found
 [here](https://github.com/llvm/torch-mlir/blob/6403c0e56f0e93e231df1c8d3dc78df7dd721b80/python/torch_mlir/__init__.py#L19)
-and includes linalg on tensors, mhlo, and tosa.
+and includes `linalg-on-tensors`, `mhlo`, and `tosa`.
```

Comment:
Can probably drop the `/blob/*/` part of this URL and change to just https://github.com/llvm/torch-mlir/blob/main/python/torch_mlir/__init__.py, then change 'mhlo' to 'stablehlo'

Reply:
Done!